plotting/density_pressure.py

```diff
@@ -0,0 +1,105 @@
+"""
+Makes a rho-P plot. Uses the swiftsimio library.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from swiftsimio import load
+
+from unyt import mh, cm, kb, K
+from matplotlib.colors import LogNorm
+from matplotlib.animation import FuncAnimation
+
+# Constants; these could be put in the parameter file but are rarely changed.
+density_bounds = [10 ** (-9.5), 1e6]  # in nh/cm^3
+pressure_bounds = [10 ** (-4), 10 ** (10)]  # in K/cm^2
+bins = 256
+
+# Plotting controls
+plt.style.use("mnras.mplstyle")
+
+
+def get_data(filename):
+    """
+    Grabs the data (P in Kelvin / cm^3 and density in mh / cm^3).
+    """
+
+    data = load(filename)
+
+    density_factor = float(data.gas.densities.cosmo_factor.a_factor)
+    pressure_factor = float(data.gas.pressures.cosmo_factor.a_factor)
+
+    number_density = (data.gas.densities * (density_factor / mh)).to(cm ** -3)
```

Comment:
See `density_internal_energy` for a better version of the cosmology transforms

Reply:
I had based it off the one in the rho-T script. But indeed, this one is better.